view/src/stores/conversations.ts

```diff
@@ -134,11 +135,15 @@ export const useConversationsStore = defineStore('conversations', () => {
     const res: AxiosResponse<ConversationInfo[]> = await axios.get('/api/conversations', {
       params: { contextId }
     })
-    const fetchedConversations = res.data.map((conv) => ({
-      ...conv,
-      createdAt: new Date(conv.createdAt),
-      updatedAt: new Date(conv.updatedAt)
-    }))
+    const fetchedConversations = res.data.map((conv) => {
+      const { ...rest } = conv
+
+      return {
+        ...rest,
+        createdAt: new Date(conv.createdAt),
+        updatedAt: new Date(conv.updatedAt)
+      }
+    })
```

Comment:
```suggestion
    const fetchedConversations = res.data.map((conv) => {
      return {
        ...conv,
        createdAt: new Date(conv.createdAt),
        updatedAt: new Date(conv.updatedAt)
      }
    })
```

Tu destructures conv mais ne supprime aucune de ces valeurs :D